.travis.yml

```diff
@@ -47,62 +36,145 @@ matrix:
     - os: linux
 
   include:
+    - os: linux
+      env: DIST=pangolin COMPILER=g++-4.7 CCOMPILER=gcc-4.7
+      addons:
+        apt:
+          sources: ['ubuntu-toolchain-r-test']
+          packages: ['g++-4.7']
+      compiler: gcc-4.7
 
-  # 1. Linux Clang Builds
+    - os: linux
+      env: DIST=pangolin COMPILER=g++-4.8 CCOMPILER=gcc-4.8
+      addons:
```

Comment:
Yes, but the referencing should go at the top of the file, not in individual entries...but I haven't had time to do that.

When builds fail it may only be a tiny number of entries in the matrix. Being able to snip out all of the passing tests and start the next build on only failing ones is easier when each entry is self contained. If the reference is tied to a specific entry I'd have to cut paste even more.